[2025-02-19T06:54:28.017+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T06:54:28.042+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_with_tasks.load_to_s3 manual__2025-02-19T06:54:10.703700+00:00 [queued]>
[2025-02-19T06:54:28.056+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_with_tasks.load_to_s3 manual__2025-02-19T06:54:10.703700+00:00 [queued]>
[2025-02-19T06:54:28.057+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-19T06:54:28.074+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): load_to_s3> on 2025-02-19 06:54:10.703700+00:00
[2025-02-19T06:54:28.090+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_with_tasks', 'load_to_s3', 'manual__2025-02-19T06:54:10.703700+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmprazc3r_5']
[2025-02-19T06:54:28.096+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1977) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-19T06:54:28.097+0000] {standard_task_runner.py:105} INFO - Job 29: Subtask load_to_s3
[2025-02-19T06:54:28.098+0000] {standard_task_runner.py:72} INFO - Started process 1979 to run task
[2025-02-19T06:54:28.171+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline_with_tasks.load_to_s3 manual__2025-02-19T06:54:10.703700+00:00 [running]> on host 27a564c12979
[2025-02-19T06:54:28.663+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline_with_tasks' AIRFLOW_CTX_TASK_ID='load_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-02-19T06:54:10.703700+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-19T06:54:10.703700+00:00'
[2025-02-19T06:54:28.666+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T06:54:28.701+0000] {logging_mixin.py:190} WARNING - /opt/***/dags/etl_pipeline.py:82 FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
[2025-02-19T06:54:31.320+0000] {load_s3.py:36} INFO - DataFrame successfully uploaded to s3://datafueler-bucket/Processed_Data/fuel_consumption_data.csv
[2025-02-19T06:54:31.324+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-19T06:54:31.336+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T06:54:31.337+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_with_tasks, task_id=load_to_s3, run_id=manual__2025-02-19T06:54:10.703700+00:00, execution_date=20250219T065410, start_date=20250219T065428, end_date=20250219T065431
[2025-02-19T06:54:31.408+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T06:54:31.429+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-19T06:54:31.433+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
